crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -589,14 +590,16 @@ impl Room {
     /// Add a new timeline slice to the timeline streams.
     #[cfg(feature = "experimental-timeline")]
     pub async fn add_timeline_slice(&self, timeline: &TimelineSlice) {
+        use tracing::warn;
+
         if timeline.sync {
             let mut streams = self.forward_timeline_streams.lock().await;
             let mut remaining_streams = Vec::with_capacity(streams.len());
             while let Some(mut forward) = streams.pop() {
                 if !forward.is_closed() {
                     if let Err(error) = forward.try_send(timeline.clone()) {
                         if error.is_full() {
-                            tracing::warn!("Drop timeline slice because the limit of the buffer for the forward stream is reached");
+                            warn!("Drop timeline slice because the limit of the buffer for the forward stream is reached");
```

Comment:
These could also add a bit more info, for which room are we dropping the slice?
```suggestion
                            warn!("Dropping timeline slice because the limit of the buffer for the forward stream is reached");
```